src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -580,7 +580,10 @@ def delete_resource(resource_ids=None, resource_group_name=None,
             results.append(operation.result())
 
     for _, id_dict in to_be_deleted:
-        logger.warning(id_dict['exception'])
+        logger.error(id_dict['exception'])
```

Comment:
Can you show me an example what this will eventually look like?

Reply:
![capture](https://user-images.githubusercontent.com/8580552/30936913-2cd140d2-a38a-11e7-8e3f-5e56c8861157.JPG)

Reply:
Fix the error message to make it more readable.

Reply:
![capture](https://user-images.githubusercontent.com/8580552/30939338-ef514178-a391-11e7-9a76-e1b3bc01dee0.JPG)

Reply:
Blue text is debug msgs